test/jdk/sun/java2d/cmm/ColorConvertOp/ColConvCCMTest.java

```diff
@@ -57,10 +57,10 @@ public class ColConvCCMTest extends ColConvTest {
     final static double [] ACCURACY = {
     // Accuracy for color conversions
         2.5,        // sRGB
-        (isOpenProfile() ? 45.0 : 10.1), // LINEAR_RGB
+        45.0,       // LINEAR_RGB
         10.5,       // GRAY
-        (isOpenProfile() ? 215.0 : 45.5), // PYCC
-        (isOpenProfile() ? 56.0 : 47.5) // CIEXYZ
+        215.0,      // PYCC
+        56.0        // CIEXYZ
```

Comment:
All this change will delete all information about "old and good" values and start to use lcms thresholds for all profiles? I think it should be the opposite - use the good data for all except lcms(or any other not that good as jdk8 profiles), like we do it now.

Reply:
Actually the purpose of these particular tests is validation of the quality of the built-in color profiles.

Reply:
I agree, I reverted almost all changes except the ones in tolerance for non-LCMS profiles. That values need to be updated to make the test pass on JDKs (e.g. JDK 8u, etc) where non-LCMS profiles are used

Reply:
Could you please clarify what is the root cause of the problem. I assume you did not update the profiles itself, so what is the problem in the new lcms library?

Reply:
As far as I know there is no any issues with the new LCMS library. There was a bug in LCMS: it used to set ‘lcms’ as profile ID to the header. As a result we used lcms threshold even for non-LCMS (kcms, etc.) profiles in the test.
In LCMS 2.16 the problem with profile ID was fixed, see https://github.com/openjdk/jdk/pull/17382/files#diff-738d14c3b278e6d7297dbc4943e90cef33fc04d61eba085218a1229c92ea9a33R941
And the tests started failing for non-LCMS profiles.